src/plonk/verifier.rs

```diff
@@ -64,7 +79,7 @@ impl<'a, C: CurveAffine> Proof<C> {
 
         // This check ensures the circuit is satisfied so long as the polynomial
         // commitments open to the correct values.
-        self.check_hx(params, vk, x_0, x_1, x_2, x_3)?;
+        self.check_hx(params, vk, beta, gamma, x_2, x_3, theta)?;
 
         // Hash together all the openings provided by the prover into a new
         // transcript on the scalar field.
```

Comment:
We need to hash the openings for the lookup argument into the transcript like we do with the permutation arguments (expand the code just below this comment).